app/src/main/java/com/sygic/travel/sdkdemo/auth/GoogleSignIn.kt

```diff
@@ -0,0 +1,100 @@
+package com.sygic.travel.sdkdemo.auth
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import com.google.android.gms.auth.api.Auth
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions
+import com.google.android.gms.auth.api.signin.GoogleSignInResult
+import com.google.android.gms.common.ConnectionResult
+import com.google.android.gms.common.api.GoogleApiClient
+import com.sygic.travel.sdkdemo.R
+
+
+class GoogleSignIn(
```

Comment:
I do not understand much what this is about. Some extracted funcitonality. Not nice, but I don't care so much for the demo app.

Reply:
I extracted these two components because each of them has around 100+ lines of code. You suggest that I should inline them into `AuthActivity` ?